### DISCUSSION TITLE
Adguard has been obstructing the Hana Banking app recently，Please handle it as soon as possible

### DISCUSSION THREAD

**Discussion by ever88** [List-KR/community#19] [General]
Adguard has been obstructing the Banking app recently, making it inconvenient for me。

specifically， the restoration of the DNS resolution for mbp.hanabank.com, and the subsequent re-establishment of the banking application's functionality. 

Please resolve it as soon as possible

**Comment by piquark6046** [comment]
Please refer https://github.com/List-KR/List-KR/discussions/732#discussioncomment-6292291.
